advocacy_docs/supported-open-source/patroni/03-rhel8_quick_start.mdx

```diff
@@ -0,0 +1,277 @@
+$ sudo systemctl disable postgresql-14
+```
+
+Patroni will bootstrap (create) the initial PostgreSQL cluster and be in charge of starting the service.
+
+Don't forget to configure the firewall if needed:
+
+```bash
+$ sudo firewall-cmd --quiet --permanent --add-service=postgresql
+$ sudo firewall-cmd --quiet --reload
+```
+
+### Watchdog
+
+Patroni will be the component interacting with the watchdog device. Set the permissions of the software watchdog on both **pg-patroni1** and **pg-patroni2** hosts:
+
+```bash
+$ cat <<EOF | sudo tee /etc/udev/rules.d/99-watchdog.rules
+KERNEL=="watchdog", OWNER="postgres", GROUP="postgres"
+EOF
+$ sudo sh -c 'echo "softdog" >> /etc/modules-load.d/softdog.conf'
+$ sudo modprobe softdog
+$ sudo chown postgres: /dev/watchdog
+```
+
+### Patroni
+
+On both **pg-patroni1** and **pg-patroni2** hosts, install Patroni and its dependencies for etcd, following the [installation guide](02-installation/).
+
+Let us now define the Patroni configuration in `/etc/patroni.yml`:
+
+```bash
+$ CLUSTER_NAME="demo-cluster-1"
+$ MY_NAME=$(hostname --short)
+$ MY_IP=$(hostname -I | awk ' {print $1}')
+$ cat <<EOF | sudo tee /etc/patroni.yml
+scope: $CLUSTER_NAME
+namespace: /db/
+name: $MY_NAME
+
+restapi:
+  listen: "0.0.0.0:8008"
+  connect_address: "$MY_IP:8008"
+  authentication:
+    username: patroni
+    password: mySupeSecretPassword
+
+etcd3:
+    hosts:
+    - etcd1:2379
+    - etcd2:2379
+    - etcd3:2379
+
+bootstrap:
+  dcs:
+    ttl: 30
+    loop_wait: 10
+    retry_timeout: 10
+    maximum_lag_on_failover: 1048576
+    postgresql:
+      use_pg_rewind: true
+      use_slots: true
+      parameters:
+        archive_mode: "on"
+        archive_command: "/bin/true"
+
+  initdb:
+  - encoding: UTF8
+  - data-checksums
+  - auth-local: peer
+  - auth-host: scram-sha-256
+
+  pg_hba:
+  - host replication replicator 0.0.0.0/0 scram-sha-256
+  - host all all 0.0.0.0/0 scram-sha-256
+
+  # Some additional users which needs to be created after initializing new cluster
+  users:
+    admin:
+      password: admin%
+      options:
+        - createrole
+        - createdb
+
+postgresql:
+  listen: "0.0.0.0:5432"
+  connect_address: "$MY_IP:5432"
+  data_dir: /var/lib/pgsql/14/data
+  bin_dir: /usr/pgsql-14/bin
+  pgpass: /tmp/pgpass0
+  authentication:
+    replication:
+      username: replicator
+      password: confidential
+    superuser:
+      username: postgres
+      password: my-super-password
+    rewind:
+      username: rewind_user
+      password: rewind_password
+  parameters:
+    unix_socket_directories: '/var/run/postgresql,/tmp'
+
+watchdog:
+  mode: required
+  device: /dev/watchdog
+  safety_margin: 5
+
+tags:
+  nofailover: false
+  noloadbalance: false
+  clonefrom: false
+  nosync: false
+EOF
+```
+
+Finally, create the [systemd file](https://github.com/zalando/patroni/blob/master/extras/startup-scripts/patroni.service) and start the Patroni service:
```

Comment:
The Rocky rpm already has a service file:
```
# rpm -ql patroni | grep service$
/usr/lib/systemd/system/patroni.service
```
Don't have an Ubuntu handy but wouldn't be surprised if it also had the service file included in the deb package

Reply:
There's an issue with that: where you install the Patroni binaries and where you put the configuration file.
In the package provided, it uses `ExecStart=/usr/bin/patroni /etc/patroni/patroni.yml` while most of the documented commands use `/etc/patroni.yml`. (The package doesn't even create the `/etc/patroni` directory)

And it also depends on how you install Patroni. With the package or with pip. I think it is better to let the user check that and adjust it if needed. Probably adding some wording like "_create the systemd file if not done by the installation package already_" would be enough...

Reply:
wording added in 5bf5f28

Reply:
Also a good reason why to let that there: EPAS. You need to customize the system user used for EPAS (enterprisedb) as shown in the debian quick-start in 193743d

Reply:
When dealing with systemd, it's better to place any of the configuration changes in /etc/systemd/... 
With the example above, an upgrade of Patroni with rpm will overwrite the unit file in /usr/lib/systemd/system.

Reply:
ok, updated in [3ae1f69](https://github.com/EnterpriseDB/docs/pull/2812/commits/3ae1f695aecd1ff7b89b1e82c83034ba2f83be8f)